src/components/Toast.js

```diff
@@ -69,7 +70,6 @@ class Toast extends Component {
     bodyClassName: null,
     progressClassName: null,
     updateId: null,
-    role: 'alert'
```

Comment:
I set the default to `alert` in the `ToastContainer`, so it seemed redundant to have another default in `Toast`. Please let me know if this assumption is incorrect.

Reply:
I completely agree with you